### PR TITLE
loss function plot 

### PR DESCRIPTION
**Pull request by hjh17**
This is a pull request for #2 

After each training process, a loss function is plotted for 

- **dev set** 
- **training set** 

Furthermore, the **accuracy** on the dev set is shown after each epoch (shown in red on the figure)

loss_function

### CODE REVIEW

**Comment by hea21** [comment]
Approved
